WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -1881,6 +1902,9 @@ nick_the_black = {
 			trait = excommunicated
 			trait = impaler
 			trait = bad_priest_christian
+			trait = cannibal_trait
+			has_character_modifier = cannibal
+			has_character_modifier = crazy_cannibal
```

Comment:
That is unrelated with the others. This is a specific nickname about the Church.